lab_6_pipeline/pipeline.py

```diff
@@ -128,11 +257,22 @@ def convert_pos(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Extracts and converts POS from the OpenCorpora tags into the UD format
         """
+        for oc_tag, ud_tag in self._tag_mapping['POS'].items():
+            if oc_tag in tags:
+                return ud_tag
+        return 'UNKN'
 
     def convert_morphological_tags(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Converts the OpenCorpora tags into the UD format
         """
+        ud_tags = {}
+        for category in ('Case', 'Number', 'Gender', 'Animacy'):
```

Comment:
1. magic constants
2. inspect if these properties can be access directly from OpenCorporaTag

Reply:
reworked